src/PerfView/App.cs

```diff
@@ -701,16 +708,18 @@ public static SymbolReader GetSymbolReader(string etlFilePath = null, SymbolRead
             ret.SourcePath = sourcePath;
             ret.Options = symbolFlags;
 
+#if !PERFVIEW_COLLECT
             if (!AppLog.InternalUser && !App.CommandLineArgs.TrustPdbs)
             {
                 ret.SecurityCheck = delegate (string pdbFile)
                 {
-                    var result = MessageBox.Show("Found " + pdbFile + " in a location that may not be trustworthy, do you trust this file?",
-                        "Security Check", MessageBoxButton.YesNo);
-                    return result == MessageBoxResult.Yes;
+                    var result = System.Windows.MessageBox.Show("Found " + pdbFile + " in a location that may not be trustworthy, do you trust this file?",
```

Comment:
:bulb: It would be preferable to place the `using` directives in conditional block instead of fully-qualifying everything.

Reply:
This kind of things is a bit of a tossup, but I don't mind if code that really is taking a bad dependency (e.g. taking a GUI dependency when I would rather App.cs be close to gui-free) be more explicit.    Basically I would like to move the code the direction of not having the ifdefs at all, but in the mean time making it slightly more painful to add more GUI dependencies is not a bad thing.

Reply:
Ironically, the reason I prefer the `using` directives is the same as the reason you prefer to not have them. 😆 I often "get a feel" for the amount a conditional dependency is used by deleting its using directive, but that doesn't work when things are fully-qualified.

Reply:
And yet, you can achieve your effect of removing the using with a simple find string on the prefix.   In the end it seems like something that is not worth mandating one way or the other.